src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageDependencies.cs

```diff
@@ -163,6 +176,55 @@ protected override void ExecuteCore()
             ReadProjectFileDependencies();
             RaiseLockFileTargets();
             GetPackageAndFileDefinitions();
+
+            FixUpNetStandardLibraryFacadeAssemblies();
+        }
+
+        /// <remarks>
+        /// The NETStandard.Library package (starting from version 2.0) adds references
+        /// to most assemblies via a .targets file rather than going through the normal
+        /// process to resolve assemblies from packages. We need to find these and
+        /// create the appropriate file definitions and dependencies to associate them
+        /// with the package.
+        /// Here we find the "facade" references from the .NETStandard.Library package,
+        /// update the existing FileDefinition, and create a FileDependency between the
+        /// assembly and the package.
+        /// </remarks>
+        private void FixUpNetStandardLibraryFacadeAssemblies()
+        {
+            var facades = References.Where(item => item.GetBooleanMetadata("Facade") == true
```

Comment:
> Why wouldn't netstandard.dll have the facade metadata?

Because this is the thing that provides all the API to a netstandard 2.0 library, not just a bunch of type forwarders for compat with other targets. Facade indicates the latter.

It should not be tied to NetStandard.Library. The package is using targets because it has different rules than nuget defaults about which TFMs actually consume its assets, so it uses targets to add references instead of the default nuget rules based on folders. Another package may come along and have the same requirement

We should reparent any Reference with NuGetPackageId under the corresponding package. Facade==true should be strictly about de-emphasizing or hiding facades should we decide to do so.